aggregator/src/aggregator/error.rs

```diff
@@ -144,6 +144,9 @@ pub enum Error {
     /// An error occurred when trying to ensure differential privacy.
     #[error("differential privacy error: {0}")]
     DifferentialPrivacy(VdafError),
+    /// the http client disconnected while we were still executing
```

Comment:
```suggestion
    /// The http client disconnected while we were still executing.
```

(nit, matching the style of other doccomments)